pkg/networkpath/traceroute/runner/runner.go

```diff
@@ -182,7 +182,11 @@ func (r *Runner) runTCP(cfg config.Config, hname string, target net.IP, maxTTL u
 	return pathResult, nil
 }
 
-func (r *Runner) processResults(res *common.Results, protocol payload.Protocol, hname string, destinationHost string, destinationPort uint16, destinationIP net.IP) (payload.NetworkPath, error) {
+func (r *Runner) processResults(res *common.Results, protocol payload.Protocol, hname string, destinationHost string) (payload.NetworkPath, error) {
+	if res == nil {
+		return payload.NetworkPath{}, fmt.Errorf("no results to process")
```

Comment:
Should this really be an error?

Reply:
Yeah that's a fair point, I'll have this return a nil error.